emoji-copy@felipeftn/stylesheet.css

```
@@ -5,20 +5,18 @@
   color: #eeeeec;
   border-radius: 32px;
   padding: 12px;
-  border: 1px solid #282828;
 }
 
 .EmojisCategory:hover,
 .EmojisCategory:focus {
-  background-color: rgba(120, 120, 120, 1);
+  background-color: #1c1c1c;
```

Here I would just change this small thing...
```suggestion
  background-color: rgba(120, 120, 120, 1);
```